gridstatus/tests/test_ercot_api.py

```diff
@@ -1072,6 +1072,37 @@ def test_get_dam_load_and_gen_60_day_resources_as_offers(self):
             assert df["Interval Start"].min() == start_date
             assert df["Interval End"].max() == end_date
 
+            assert df.groupby(["Interval Start", "Resource Name"]).size().max() == 1
+
+    def test_get_dam_load_and_gen_60_day_resources_as_offers_repeated_offers(self):
```

Comment:
I did not use VCR for this test because the casette `yaml` file was 10.7 MB on disk and, once that file is added to the repo, it can never be removed from the repo history.